src/IterablePushPlatform.ts

```diff
@@ -1,10 +1,7 @@
-
 enum IterablePushPlatform {
```

Comment:
Maybe have export here to align with other files?

Reply:
Good catch!